samples/goroutines2.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"fmt"
+	"time"
+	"runtime"
+)
+
+func say(s string) {
+	for i := 0; i < 5; i++ {
+		time.Sleep(100 * time.Millisecond)
+		fmt.Println(s)
+	}
+}
+
+func main() {
+	fmt.Println("Number of logical processors:", runtime.NumCPU())
+	go say("world")
+	say("hello")
```

Comment:
you should probably sleep a little more at the end to ensure the both statements get printed.
Also: what's the point of telling us the `NumCPU()`

Reply:
I didn't get the first part of your comment. Where should I put the sleep ? I thought all the lines are getting printed.

`NumCPU()` is directly linked to the number of OS Threads (used by default) for the program. So I thought it might be useful for users to see the output of what goplay provides vs. what's there on their local machines. And they can play around by setting GOPROCMAX and to see how this gets reset / changed.

Reply:
After we had a talk about the advanced stuff etc. I've removed this line from the code now.